### ISSUE TITLE
Exception parsing MQTT event: Field "data" of type ServiceEventChargingData

### ISSUE THREAD

**Issue by sonar98** [skodaconnect/homeassistant-myskoda#78]
With version 1.4.5 I see some warnings in the logs.

````
Logger: myskoda.mqtt
Bron: runner.py:189
Eerst voorgekomen: 7 oktober 2024 om 18:18:56 (7 gebeurtenissen)
Laatst gelogd: 7 oktober 2024 om 18:33:26

Exception parsing MQTT event: Field "data" of type ServiceEventChargingData in ServiceEventCharging has invalid value {'mode': 'off', 'state': 'notReadyForCharging', 'soc': '71', 'chargedRange': '288', 'timeToFinish': 'null', 'userId': 'xxxxxxxx, 'vin': 'TMxxxxxx'}

**Comment by WebSpider** [comment]
This issue will be solved in the next release :partying_face:

**Comment by WebSpider** [comment]
This has been fixed in v1.5.0
